scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -106,20 +106,25 @@ class RunCoverage(
   fun execute() {
     val testFileExemptionList = loadTestFileExemptionsProto(testFileExemptionTextProto)
       .testFileExemptionList
-      .filter { it.testFileNotRequired }
+      .filter { it.testFileNotRequired || it.sourceFileIsIncompatibleWithCodeCoverage }
       .map { it.exemptedFilePath }
 
     if (filePath in testFileExemptionList) {
       println("This file is exempted from having a test file; skipping coverage check.")
```

Comment:
Note to self: I should update this to have a different warning when exempted due to incompatibility (to be clearer and not incorrect since the actual reason for exemption is different in those cases).